ui/packages/catalog-ui-search/src/main/webapp/component/select/select.collection.view.js

```diff
@@ -170,6 +170,18 @@ module.exports = Marionette.CollectionView.extend({
     str = str.toString()
     return this.options.matchcase === true ? str : str.toLowerCase()
   },
+  getWords: function(str) {
+    //Handle camelcase
+    str = str.replace(/([A-Z])/g, ' $1')
```

Comment:
This goes against the conventions for our taxonomy names, but would this cause an attribute named `EXAMPLE` to not appear as a result when I user typed in `EXA`?

Reply:
This is a good catch! Since users can actually input their own text for some text boxes this will break for that case.
![screen shot 2019-01-28 at 10 15 04](https://user-images.githubusercontent.com/9013407/51853485-a6a1fd80-22e5-11e9-9a13-ade1992f913c.png)
![screen shot 2019-01-28 at 10 15 13](https://user-images.githubusercontent.com/9013407/51853492-a7d32a80-22e5-11e9-98be-f5580950b315.png)